sdk/servicebus/service-bus/package.json

```diff
@@ -87,24 +100,26 @@
     "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "node .scripts/prepSamples.js && cd samples && tsc -p .",
-    "build:test": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
-    "build": "tsc -p . && rollup -c 2>&1",
+    "build:test:browser": "tsc -p . && cross-env BROWSER_TEST=true rollup -c 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
```

Comment:
rollup.test.config.js doesn't seem to look at the ONLY_NODE / ONLY_BROWSER / BROWSER_TEST environment variables. You can probably remove `cross-env ONLY_NODE=true` here.

Reply:
Agree, will look into it.